app/lib/bk/compat/parsers/circleci/orbs/docker/check.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module BK
+  module Compat
+    module CircleCISteps
+      # Handles the translation of docker/check command
+      class DockerOrb
+        def translate_docker_check(config)
+          docker_password = config.fetch('docker-password', 'DOCKER_PASSWORD')
+          docker_username = config.fetch('docker-username', 'DOCKER_LOGIN')
+          registry = config.fetch('registry', 'docker.io')
+
+          [
+            (if config.fetch('use-docker-credentials-store', false)
+               [
+                 translate_docker_install_docker_credential_helper({ 'arch' => config.fetch('arch', 'amd64') }),
```

Comment:
if you only want a particular value if included in the original hash you can use `slice`:

```suggestion
                 translate_docker_install_docker_credential_helper(config.slice('arch')),
```